pkg/wasm/convert.go

```diff
@@ -68,6 +101,15 @@ func convert(resource *anypb.Any, cache Cache) (newExtensionConfig *anypb.Any, s
 		wasmConfigConversionCount.
 			With(resultTag.Value(status)).
 			Increment()
+
+		if newExtensionConfig == resource && !sendNack && status != noRemoteLoad {
+			var err error
+			newExtensionConfig, err = createAllowAllFilter(ec.GetName())
```

Comment:
wondering why cannot just ignore the wasm filter

Reply:
If an ECDS for a corresponding filter is not passed to Envoy, the relevant routes will be blocked.

Reply:
IC, we are not rewriting on the initial xds response.